cmd/minikube/cmd/start.go

```diff
@@ -336,6 +338,14 @@ func clusterUp(config *cluster.MachineConfig) {
 	}
 }
 
+func validateOpenshiftVersion() {
+	if viper.IsSet(openshiftVersion) {
+		if !util.ValidateOpenshiftMinVersion(viper.GetString(openshiftVersion)) {
+			glog.Exitf("%s is not supported with minishift", viper.GetString(openshiftVersion))
```

Comment:
Also, let's be a not more verbose: "Minishift does not support Openshift in  version %s. You need to use a version >= 1.3.1.